packages/fxa-auth-server/scripts/paypal-processor.ts

```diff
@@ -13,6 +15,21 @@ import { setupProcessingTaskObjects } from '../lib/payments/processing-tasks-set
 
 const pckg = require('../package.json');
 const config = require('../config').getProperties();
+const PAYPAL_PROCESSOR_LOCK = 'fxa-paypal-processor-lock';
+const DEFAULT_LOCK_DURATION_MS = 10000;
```

Comment:
10 seconds is a bit low for a default, given we only run it hourly at max, 5 minutes should be enough even with clock drift to prevent dupe runs.